### DISCUSSION TITLE
Van Cone - Feedback requested - 1.0 release candidate!

### DISCUSSION THREAD

**Discussion by b-rad-c** [vanjs-org/van#254] [General]
Hello all!

I am the author of the Van Cone SPA framework for VanJS. I have been working on this framework for a couple of months now and am about ready to take it out of beta mode which means the API will no longer have breaking changes going forward.

I'm taking feedback for any possible questions/comments/issues regarding the API as it stands right now. I'm happy to take ideas for new features, however I'm mainly looking for potentially issues with the API in terms of backward compatibility should we implement new features.

We've added typescript support thanks to @yahia-berashish 

There's a new `route` function to declare routes, it's similar to what we discussed on [this thread](https://github.com/vanjs-org/van/discussions/185) although it does not support the outlet function.

The new version is available on npm, I have changed the repository to [this location](https://github.com/medium-tech/van-cone), with the next release I will update the current source in the van repo w a readme file pointing to the new location.

And I have a scrappy website hosted on GitHub, [available here](https://medium-tech.github.io/van-cone-website/).

Thanks for all the feedback so far, looking forward to more ideas, I have more features in mind that I'll be releasing over the coming months.

**Comment by Tao-VanJS** [comment]
This is great! I love the website and Van Cone logo.

**Reply by b-rad-c** [reply]
Thanks! I will pass along your compliments of the logo to ChatGPT 😂

**Comment by b-rad-c** [comment]
Already made one more change. The `isCurrentPage` function will now return a boolean instead of a van derive object that returns a boolean. This means users can do `isCurrentPage('home') === true` instead of `isCurrentPage('home').val === true`
